docs/book/src/reference/commands/alpha_update.md

```diff
@@ -113,7 +153,7 @@ Incoming changes
 
 ## Commit message used in `--squash` mode
 
-> [kubebuilder-automated-update]: update scaffold from <from> to <to>; (squashed 3-way merge)
+> chore(kubebuilder): update scaffold from <from> to <to>
```

Comment:
we cannot use chore, we do not know if is a breaking change your not 
I think we should only use `[kubebuilder-automated-update]: update scaffold from <from> to <to>; (squashed 3-way merge)`